package.json

```diff
@@ -19,29 +19,31 @@
   ],
   "sideEffects": false,
   "dependencies": {
-    "react": ">=0.14.x",
-    "styled-components": ">=2.4.0"
+    "@babel/runtime": "7.11.2",
+    "react": ">=16.13.x",
+    "styled-components": ">=5.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.1.2",
-    "@babel/plugin-external-helpers": "^7.2.0",
-    "@babel/plugin-transform-runtime": "^7.2.0",
-    "@babel/preset-env": "^7.2.3",
-    "@babel/preset-react": "^7.0.0",
-    "@types/react": "^16.0.34",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "jest": "^21.2.1",
-    "prop-types": "^15.6.0",
-    "raf": "^3.4.0",
-    "react-dom": "^16.2.1",
-    "rollup": "^0.68.1",
-    "rollup-plugin-babel": "^4.0.3",
-    "rollup-plugin-commonjs": "9.2.0",
-    "rollup-plugin-node-resolve": "^4.0.0",
-    "rollup-plugin-replace": "^2.1.0",
-    "rollup-plugin-size-snapshot": "^0.7.0",
-    "rollup-plugin-uglify": "^6.0.0"
+    "@babel/core": "^7.11.6",
+    "@babel/plugin-external-helpers": "^7.10.4",
+    "@babel/plugin-transform-runtime": "^7.11.5",
+    "@babel/preset-env": "^7.11.5",
+    "@babel/preset-react": "^7.10.4",
+    "@rollup/plugin-babel": "5.2.1",
+    "@rollup/plugin-commonjs": "15.1.0",
+    "@rollup/plugin-node-resolve": "9.0.0",
+    "@rollup/plugin-replace": "2.3.3",
+    "@types/react": "^16.9.51",
+    "babel-plugin-transform-react-remove-prop-types": "0.4.24",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.5",
+    "jest": "^26.5.2",
```

Comment:
We don't actually have any tests, we can get rid of these packages for now.

Reply:
as well as jest configuration inside package.json file?

Reply:
Yes. Currently, we don't use them. I'm considering using storybook but that'll be later.